<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayerTestForm</class>
 <widget class="QWidget" name="PlayerTestForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>667</width>
    <height>523</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::ExpandingFieldsGrow</enum>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Playback status:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="playbackStatusLbl">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>&lt;unknown&gt;</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Loop status:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QLabel" name="loopStatusLbl">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>&lt;unknown&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="loopStatusCombo">
           <property name="editable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="loopStatusSetBtn">
           <property name="text">
            <string>Set</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Shuffle:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <widget class="QLabel" name="shuffleLbl">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>&lt;unknown&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="shuffleOnBtn">
           <property name="text">
            <string>On</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="shuffleOffBtn">
           <property name="text">
            <string>Off</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Volume:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_12">
         <item>
          <widget class="QLabel" name="volumeLbl">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>&lt;unknown&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="volumeSpinBox">
           <property name="minimum">
            <double>-9999.989999999999782</double>
           </property>
           <property name="maximum">
            <double>9999.989999999999782</double>
           </property>
           <property name="singleStep">
            <double>0.050000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="volumeSetBtn">
           <property name="text">
            <string>Set</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Maximum rate:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="maxRateLbl">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>&lt;unknown&gt;</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Minimum rate:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLabel" name="minRateLbl">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>&lt;unknown&gt;</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Rate:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_13">
         <item>
          <widget class="QLabel" name="rateLbl">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>&lt;unknown&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line_4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="rateSpinBox">
           <property name="minimum">
            <double>-9999.989999999999782</double>
           </property>
           <property name="maximum">
            <double>9999.989999999999782</double>
           </property>
           <property name="singleStep">
            <double>0.250000000000000</double>
           </property>
           <property name="value">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="rateSetBtn">
           <property name="text">
            <string>Set</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Last received position:</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QLabel" name="lastKnownPosLbl">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>&lt;unknown&gt;</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Predicted position:</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QLabel" name="estPosLbl">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>&lt;unknown&gt;</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Can control:</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QLabel" name="canControlLbl">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>&lt;unknown&gt;</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Can play:</string>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QLabel" name="canPlayLbl">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>&lt;unknown&gt;</string>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Can pause:</string>
         </property>
        </widget>
       </item>
       <item row="11" column="1">
        <widget class="QLabel" name="canPauseLbl">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>&lt;unknown&gt;</string>
         </property>
        </widget>
       </item>
       <item row="12" column="0">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Can go previous:</string>
         </property>
        </widget>
       </item>
       <item row="12" column="1">
        <widget class="QLabel" name="canGoPrevLbl">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>&lt;unknown&gt;</string>
         </property>
        </widget>
       </item>
       <item row="13" column="0">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Can go next:</string>
         </property>
        </widget>
       </item>
       <item row="13" column="1">
        <widget class="QLabel" name="canGoNextLbl">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>&lt;unknown&gt;</string>
         </property>
        </widget>
       </item>
       <item row="14" column="0">
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>Can seek:</string>
         </property>
        </widget>
       </item>
       <item row="14" column="1">
        <widget class="QLabel" name="canSeekLbl">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>&lt;unknown&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>Metadata</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableView" name="metadataTableView">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QPushButton" name="playBtn">
       <property name="text">
        <string>Play</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pauseBtn">
       <property name="text">
        <string>Pause</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="playPauseBtn">
       <property name="text">
        <string>Play/Pause</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stopBtn">
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="prevBtn">
       <property name="text">
        <string>Previous</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="nextBtn">
       <property name="text">
        <string>Next</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLineEdit" name="openUriEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="openUriBtn">
       <property name="text">
        <string>Open URI</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_5">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="seekSpinBox">
       <property name="suffix">
        <string/>
       </property>
       <property name="minimum">
        <number>-999999999</number>
       </property>
       <property name="maximum">
        <number>999999999</number>
       </property>
       <property name="singleStep">
        <number>1000000</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="seekTimeUnits">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="seekBtn">
       <property name="text">
        <string>Seek by</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QLabel" name="label_32">
       <property name="text">
        <string>Track id:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="setPosTrackIdEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="label_33">
       <property name="text">
        <string>Position:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="setPosSpinBox">
       <property name="suffix">
        <string/>
       </property>
       <property name="minimum">
        <number>-999999999</number>
       </property>
       <property name="maximum">
        <number>999999999</number>
       </property>
       <property name="singleStep">
        <number>1000000</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="setPosTimeUnits">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="setPosBtn">
       <property name="text">
        <string>Set position</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
